unity-renderer/Assets/DCLPlugins/BuilderInWorld/Scripts/BuiilderAPI/BuilderAPIController.cs

```diff
@@ -269,7 +305,8 @@ public Promise<Manifest> GetManifestByCoords(string landCoords)
                     manifest = JsonConvert.DeserializeObject<Manifest>(result);
                     fullNewProjectPromise.Resolve(manifest);
                 }
-                else if (x["data"].First["version"] != null)
+                // Sometimes the API respond that OK but the data is empty so we need to check that 
+                else if (x["data"].HasValues && x["data"].First["version"] != null)
```

Comment:
rename `x` to a more descriptive name.. also are we safe on accessing json nodes that may not exists?

Reply:
Done! Yes, the builder API is not behaving correctly and this is a low priority for them right now so this should handle the problem until they tackle the problem.

Some times it replies with the manifest directly and sometimes it replies with a data structure, so we are checking if the data exist before trying to get the version